src/makie.jl

```diff
@@ -471,7 +471,54 @@ function figure_box_inset()
     lines!(inset_ax1, 1:10)
     scatter!(inset_ax2, 1:10, color = :black)
     fig
-end   
+end 
+
+function scatters_in_3D()
+    GLMakie.activate!() # hide 
+    Random.seed!(123)
+    xyz = randn(10, 3)
+    x, y, z = xyz[:,1], xyz[:,2], xyz[:,3]
+    fig = Figure(resolution = (1800,600), fontsize = 26)
```

Comment:
```suggestion
    fig = Figure(resolution = (1800, 600), fontsize = 26)
```
I'm doing many nitpicky suggestions. Feel free to merge without fixing all these things. We can also fix it later.